inc/c_util/slist.h

```diff
@@ -9,18 +9,18 @@
 #include <stdbool.h>
 #endif
 
-typedef struct SLIST_ENTRY_TAG
+typedef struct SINGLYLINKEDLIST_ENTRY_TAG
```

Comment:
I think you can place the definition of the structure inside the .c file to achieve the information hiding (encapsulation) aspect of object-oriented programming using the C language.

Reply:
I'm not sure which option is better.